index.js

```diff
@@ -0,0 +1,33 @@
+var geek = [];
+
+for (let i = 0; i <= 5; i++) {
+  if (i == 0) {
+    geek[i] = prompt("Enter Emplyee No: ");
```

Comment:
This is not good approach to ask for the prompt. Rather define an array of the objects
```js
const employeeStats = [
  {...},
  {...},
  {...}
]
```

Then run a loop over it and populate HTML through it